pyctcdecode/language_model.py

```diff
@@ -19,6 +21,10 @@
     LOG_BASE_CHANGE_FACTOR,
 )
 
+DEFAULT_MODEL_FILE_NAME = "kenLM.arpa"
+LIBRARY_NAME = "pyctcdecode"
```

Comment:
suggest we remove this and put it in `load_from_hf_hub`. The package name lives right next to the `__version__` anyway. So we can do 
```
from . import __version__ as VERSION, package_name as LIBRARY_NAME
```